CppCoreGuidelines.md

```diff
@@ -7152,16 +7152,16 @@ Some of these expressions are unconditionally bad (e.g., they rely on undefined
 
 **Example**:
 
-	x=k*y+z;	// OK
+	x=k*y+z;              // OK
 
-	auto t1 = k*y;	// bad: unnecessarily verbose
+	auto t1 = k*y;        // bad: unnecessarily verbose
 	x = t1+z;
 
-	if(0<=x && x<max) // OK
+	if(0<=x && x<max) {}  // OK
```

Comment:
I see. What I am trying to do is running clang or gcc on these code examples where possible, so that compiler errors and warnings can be handled where appropriate (and ignored where they should). That approach will only work well when the code examples generally compile.

With respect to the purpose of the document, I also believe it is nice if the reader is not confused. In this case, there is an invalid statement with "// OK" behind it, which can be confusing. But I can still remove if you insist.